src/utilities/templates/utilities/side_menu_items.html

```diff
@@ -0,0 +1,11 @@
+<!-- Modular Side Menu Items -->
+<div class="nav list-group modular-side-menu hidden-xs sidebar-menu" id="lg-menu-secondary">
+{% for side_menu_item in side_menu_items %}
+  <a id="{{side_menu_item.label}}-menu" class="list-group-item
+      {% if side_menu_item.url in request.path_info %}active{% endif %}"
+        href="{{ side_menu_item.url }}">
+    <i class="fa fa-{{ side_menu_item.fa_icon }} fa-fw"></i>&nbsp; {{ side_menu_item.label }}
+  </a>
+{% endfor %}
+</div>
+<a href="{% url 'utilities:menu_items' %}?is_side_menu=true" class="hidden-xs" style="text-decoration: none; color: #777;"><i class="fa fa-fw fa-pencil"></i>&nbsp;&nbsp;Customize this menu</a>
```

Comment:
Why did you opt for using get parameters?  This feels clunky to me and I think makes it difficult for future developers, since we don't use get parameters anywhere else in this project.  Even here for example, it's a hard coded addition that is prone to easy breakage and goes against the whole django url design philosophy.  This should be a view with it's own name.